test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/messaging/QueueUriProvider.java

```diff
@@ -1,14 +1,14 @@
 package uk.gov.justice.services.test.utils.core.messaging;
 
 import static java.lang.String.format;
-import static uk.gov.justice.services.test.utils.common.host.TestHostProvider.getHost;
+import static uk.gov.justice.services.test.utils.common.host.TestHostProvider.getArtemisHost;
 
 public class QueueUriProvider {
 
     private static final String BASE_URI_PATTERN = "tcp://%s:61616";
 
     public String getQueueUri() {
-        return format(BASE_URI_PATTERN, getHost());
+        return format(BASE_URI_PATTERN, getArtemisHost());
     }
```

Comment:
I'm afraid you can't change this - there are a lot of integration tests that rely on getting the correct url to speak to the deployed app. You'll need to create another class or method